Tasks/PackerBuild/task.json

```diff
@@ -103,18 +103,58 @@
             "groupName": "AzureDetails"
         },
         {
+            "name": "baseImageSource",
+            "type": "pickList",
+            "label": "Base image source",
+            "required": true,
+            "defaultValue": "default",
+            "helpMarkDown": "Select the source for base image.",
+            "properties": {
+                "EditableOptions": "True"
+            },
+            "options": {
+                "default": "Builtin",
```

Comment:
rhs has been re-named as "Gallery" after review with Atul. And in my opinion, "default" is better than "gallery" from code perspective